src/components/ImportLogsDetail.vue

```diff
@@ -0,0 +1,281 @@
+            <ion-icon slot="start" :icon="documentTextOutline" />
+            <ion-label>
+              <p class="overline">{{ log.logId }}</p>
+              {{ log.contentName }}
+              <p>{{ getDateTime(log.createdDate) }}</p>
+            </ion-label>
+          </ion-item>
+  
+          <ion-label>
+            {{ getDateTime(log.startDateTime) }}
+            <p>{{ translate('Started') }}</p>
+          </ion-label>
+  
+          <ion-label>
+            {{ getDateTime(log.finishDateTime) }}
+            <p>{{ translate('Finished') }}</p>
+          </ion-label>
+          
+          <ion-badge color="success" v-if="log.statusId === 'SERVICE_FINISHED'">{{ translate('Finished') }}</ion-badge>
+          <ion-badge color="danger" v-if="log.statusId === 'SERVICE_FAILED'">{{ translate('Failed') }}</ion-badge>
+          <ion-badge color="success" v-if="log.statusId === 'SERVICE_RUNNING' || log.statusId === 'SERVICE_QUEUED'">{{ translate('Running') }}</ion-badge>
+          
+          <div class="ion-text-center" lines="none" v-if="log.errorRecordContentId" button @click="downloadCsv(log?.errorRecordContentId)">
+            <ion-icon slot="start" :icon="cloudDownloadOutline" />
+            <ion-label>
+              <p>{{ translate('Failed records') }}</p>
+            </ion-label>
+          </div>
+          <div v-else></div>
+  
+          <ion-button fill="clear" color="medium" @click="openDownloadLogsFilePopover(log, $event)">
+            <ion-icon slot="icon-only" :icon="ellipsisVerticalOutline" />
+          </ion-button>
+        </div>
+      </div>
+      <div v-else class="empty-state">
+        {{ translate('No logs found') }}
+      </div>
+    </ion-content>
+  </ion-page>
+</template>
+
+<script>
+import { checkmarkOutline, codeWorkingOutline, cloudDownloadOutline, documentTextOutline, ellipsisVerticalOutline, fileTrayFullOutline, globeOutline, optionsOutline, pulseOutline, shareSocialOutline, warningOutline } from "ionicons/icons";
+import { IonBackButton, IonBadge, IonButton, IonChip, IonContent, IonHeader, IonIcon, IonItem, IonLabel, IonList, IonPage, IonSpinner, IonTitle, IonToolbar, popoverController } from "@ionic/vue";
+import { defineComponent } from 'vue'
+import { mapGetters, useStore } from 'vuex'
+import { responseFileType } from '@/utils';
+import { translate } from '@hotwax/dxp-components'
+import { DateTime } from 'luxon'
+import DownloadLogsFilePopover from "@/components/DownloadLogsFilePopover.vue";
+import { JobService } from "@/services/JobService";
+import logger from '@/logger';
+
+export default defineComponent ({
+  name: "ImportLogsDetail",
+  components: {
+    IonBackButton,
+    IonBadge,
+    IonButton,
+    IonChip,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonPage,
+    IonSpinner,
+    IonTitle,
+    IonToolbar,
+  },
+  data() {
+    return {
+      configDetails: {},
+      selectedChip: 'All',
+      chips: [
```

Comment:
This should be renamed to logFilters and it should be the array of object, each filter object with key and value. So that when apply filters you can pass the ID.